catalog/ngis-cat/projects_ngis-cat_assets_DEA_NIDEM.jsonnet

```diff
@@ -0,0 +1,166 @@
+    NIDEM complements existing intertidal extent products, and provides data to 
+    support a new suite of use cases that require a more detailed understanding 
+    of the three-dimensional topography of the intertidal zone, such as 
+    hydrodynamic modelling, coastal risk management and ecological habitat 
+    mapping.
+    For more information, please see the 
+    [DEA Intertidal Elevation](https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'australia',
+    'dea_australia',
+    'dem',
+    'nidem',
+  ],
+  providers: [
+    ee.producer_provider('Digital Earth Australia', 'https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(108.81, -44.41, 157.82, -9.13, '1986-08-16T00:00:00Z', '2017-07-31T23:59:59Z'),
+  summaries: {
+    gsd: [
+      25,
+    ],
+    'eo:bands': [
+      {
+        name: 'nidem',
+        description: |||
+          Primary product: elevation in meter units relative to modelled Mean 
+          Sea Level. 
+          [Band details](https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details)
+        |||,
+        'gee:units': units.meters,
+      },
+      {
+        name: 'nidem_mask',
+        description: |||
+          Quality flag. 
+          [Band details](https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details)
+        |||,
+        'gee:classes':[
+          {
+            color: '00ff00',
+            description: 'above 25 m relative to Mean Sea Level',
+            value: 1,
+          },
+          {
+            color: 'ff0000',
+            description: 'below -25 m relative to Mean Sea Level',
+            value: 2,
+          },
+          {
+            color: 'ffff00',
+            description: 'invalid elevation estimates',
+            value: 3,
+          },
+        ],
+      },
+      {
+        name: 'nidem_uncertainty',
+        description: |||
+          Uncertainty meters. 
+          [Band details](https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details)
+        |||,
+        'gee:units': units.meters,
+      },
+      {
+        name: 'nidem_unfiltered',
+        description: |||
+          Un-cleaned elevation in meters. 
+          [Band details](https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details)
+        |||,
+        'gee:units': units.meters,
+      },
+    ],
+    nidem: {minimum: -5.0, maximum: 3.9, 'gee:estimated_range': false},
+    nidem_uncertainty: {minimum: 0, maximum: 0.3, 'gee:estimated_range': false},
+    nidem_unfiltered: {minimum: -5.0, maximum: 3.9, 'gee:estimated_range': false},
+    'gee:visualizations': [
+      {
+        display_name: 'nidem',
+        lookat: {lat: -23.70, lon: 133.88, zoom: 5},

```

Comment:
Use lon, lat to match [Map.setCenter](https://developers.google.com/earth-engine/apidocs/map-setcenter)